packages/es-components/babel.config.js

```diff
@@ -1,11 +1,32 @@
-const wtwConfig = {
-  extractFormatMessage: false,
-  transformFormatMessage: false
-};
+module.exports = function(api) {
+  const modules = api.env('test') ? 'auto' : false;
+
+  const presets = [
+    [
+      'wtw-im',
+      {
+        extractFormatMessage: false,
+        transformFormatMessage: false,
+        env: {
+          useBuiltIns: 'usage',
+          corejs: 3,
+          modules: modules
+        }
+      }
+    ]
+  ];
+
+  const plugins = [
+    ['@babel/plugin-transform-runtime', { corejs: 3 }],
+    '@babel/plugin-proposal-export-default-from',
+    '@babel/plugin-proposal-class-properties',
+    ['transform-react-remove-prop-types', { mode: 'wrap' }]
```

Comment:
Both the `class-properties` and `remove-prop-types` plugins are already included in the preset. Do we still need these two here? Are these just different versions of what we include in the preset?

Reply:
Ah, I just didn't catch that it already had them since it doesn't list them in the README. It doesn't look like we can pass settings to transform-runtime though.

Reply:
No, which is why I'm fine with keeping the `plugin-transform-runtime` plugin in here. Something we could do with the preset in the next update is being able to specify which version of corejs to use, defaulting to 3. It *should* overwrite the preset provided `corejs: 2` setting.